### ISSUE TITLE
Maven build is broken

### ISSUE THREAD

**Issue by sigpwned** [chimbori/crux#7]
The POM is missing a few properties. I opened a pull request to fix the issue: https://github.com/chimbori/crux/pull/6

Thanks for a great project in crux! :)

On the subject of maven, would the crux team consider releasing to maven central?

**Comment by chimbori-dev** [comment]
> On the subject of maven, would the crux team consider releasing to Maven Central? 

Yes! Would love that. I started it, but due to a lack of experience, motivation, and time, I didn’t finish it.

Looks like you have all three, so we’d love to accept your contributions!

BTW, I recently read https://www.niels-ole.com/ownership/2018/03/16/github-forks.html and want to un-fork this repo. This potentially means deleting and re-creating it (at the same location.) If you don’t mind, please give me a couple days to finish this task (I will prioritize it higher), and then we can set up the Maven inclusion. How does that sound?

**Comment by sigpwned** [comment]
That sounds great! Thanks for the quick response and open attitude. Drop me a line here when you’re ready, and we’ll get the Maven Central release done! 🙂
